File: Nível 1/3170 - Bolinhas de Natal/main.lua
bolinhas = tonumber(io.read())
galhos = tonumber(io.read())
calculo = math.floor(galhos / 2)

if calculo <= bolinhas or bolinhas >= galhos then 
    print("Amelia tem todas bolinhas!")
else 
    print(string.format("Faltam %d bolinha(s)", calculo - bolinhas))
end

File: Nível 1/3170 - Bolinhas de Natal/main.py
bolinhas = int(input("A "))
galhos = int(input("B "))

calculo = galhos // 2

if calculo <= bolinhas or bolinhas >= galhos:
    print("Amelia tem todas bolinhas!")
else:
    print(f"Faltam {calculo - bolinhas} bolinha(s)")